selfservice/strategy/oidc/provider.go

```diff
@@ -35,4 +35,5 @@ type Claims struct {
 	PhoneNumber         string `json:"phone_number,omitempty"`
 	PhoneNumberVerified bool   `json:"phone_number_verified,omitempty"`
 	UpdatedAt           int64  `json:"updated_at,omitempty"`
+	HD                  string `json:"hd,omitempty"`
```

Comment:
I am not a fan of simply appending new fields here as we want the Claims struct to contain only regular OIDC claims (`hd` is not such a claim). Because this will eventually conflict. Maybe `hd` is a string on Google, but an int on Amazon - and so on.

We can leave this for now as it is but this should really be something generic like `ExtraClaims` that are of `map[string]interface{}` instead.

Reply:
This makes sense to do :+1: